inc/saiport.h

```diff
@@ -1698,6 +1765,95 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SYSTEM_PORT,
 
+    /**
+     * @brief MDIX status to the user
+     *
+     * @type sai_port_mdix_mode_status_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_MDIX_MODE_STATUS,
+
+    /**
+     * @brief Configure MDIX mode
+     *
+     * @type sai_port_mdix_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MDIX_MODE_AUTO
+     */
+    SAI_PORT_ATTR_MDIX_MODE,
+
+    /**
+     * @brief Configure Auto Neg mode
+     *
+     * @type sai_port_auto_neg_config_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_AUTO_NEG_CONFIG_MODE_DISABLED
+     */
+    SAI_PORT_ATTR_AUTO_NEG_CONFIG_MODE,
+
+    /**
+     * @brief Configure power mode
+     *
+     * @type sai_port_power_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_POWER_MODE_AUTO
+     */
+    SAI_PORT_ATTR_POWER_MODE,
+
+    /**
+     * @brief Enabled auto detection between 1000X and SGMII slave mode
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_AUTO_DETECT,
+
+    /**
+     * @brief EEE latency mode. Fixed latency - False, Variable latency - True
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_EEE_LATENCY_MODE,
+
+    /**
+     * @brief Enable Auto medium to select both Fiber and Copper media types
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_AUTO_MEDIUM,
+
+    /**
+     * @brief Configure media precedence Fiber preferred -> True, Copper preferred -> False
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_MEDIA_PREFERRED,
```

Comment:
Please capture what is the expected sequence to be followed with these attribute if an application requires the medium "auto-detect" to be disabled and initialize the port in "fiber" mode by default.

Reply:
By default auto-detect will be disabled. If its enabled during prior execution, then SAI_PORT_ATTR_AUTO_DETECT attribute can be set to false to disable it. To initialize fiber ports, SAI_PORT_ATTR_MEDIA_PREFERRED should be set to true to use Fiber interface